ngrok/src/config/common.rs

```diff
@@ -209,6 +209,8 @@ pub(crate) struct CommonOpts {
     // Policy that defines rules that should be applied to incoming or outgoing
     // connections to the edge.
     pub(crate) policy: Option<Policy>,
+    // Enables ingress for ngrok endpoints.
+    pub(crate) bindings: Option<Vec<String>>,
```

Comment:
does this need to be an `Option<Vec>` or can it just be `Vec`? what's the semantic difference between `None` and `Some([])`?

Reply:
I think we can get rid of the Option here

Reply:
i think that'll really tidy up the rest of the code!